editor/src/clj/editor/attachment.clj

```diff
@@ -29,6 +29,7 @@
   ;; reorder-fn: fn of reordered-nodes -> txs
   (atom {:add {} :get {} :reorder {}}))
 
+;; SDK api
```

Comment:
Hmm. Depending on how this is used, having a `state-atom` like this will pollute the tests following a test that brings in the Spine plugin.

We have a similar issue around defmultis and protocols we expose through the SDK api, but most of those will not have any effect unless a matching node enters the graph. How is this `register!` function different?

Would it make sense to move the state to the workspace node and have the plugin supply a workspace argument before exposing this as SDK api?